pkg/utils/utils.go

```diff
@@ -170,6 +173,27 @@ func GetTrimmedGroupName(group string) string {
 	return group
 }
 
+// GetRestorePvcSizePercetage - get the percentage of size that need to increased during the kdmp restore PVC creation.
+// In the case failure, retunring zero to assume default 10% value.
+func GetRestorePvcSizePercetage() int {
+	fn := "GetRestorePvcSizePercetage"
+	kdmpData, err := core.Instance().GetConfigMap(drivers.KdmpConfigmapName, drivers.KdmpConfigmapNamespace)
+	if err != nil {
+		logrus.Warnf("%v: error in reading configMap [%v/%v]",
+			fn, drivers.KdmpConfigmapName, drivers.KdmpConfigmapNamespace)
+		return 0
+	}
+	if len(kdmpData.Data[kdmpRestorePvcSizePercetageKey]) != 0 {
+		sizePercentage, err := strconv.Atoi(kdmpData.Data[kdmpRestorePvcSizePercetageKey])
+		if err != nil {
+			logrus.Infof("%v: error in coverting kdmpRestorePvcSizePercetage to integer: %v", fn, err)
```

Comment:
```suggestion
			logrus.Warnf("%v: error in coverting kdmpRestorePvcSizePercetage to integer: %v", fn, err)
```

Shouldn't this be warn instead of info?

Reply:
Too engineering message for Info.  "error in coverting string to int"

Reply:
Changed it to Debugf.